<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ibmProveedores</class>
 <widget class="QDialog" name="ibmProveedores">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gnuOptics &gt; Proveedores &gt; Nuevo</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nuevo proveedor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imagen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../images/gnuoptics.qrc">:/gnuoptics/keko.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lblProveedor">
           <property name="text">
            <string>Nombre del proveedor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCIF">
           <property name="text">
            <string>CIF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTelefono">
           <property name="text">
            <string>Teléfono</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblEmail">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDireccion">
           <property name="text">
            <string>Dirección</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCuidad">
           <property name="text">
            <string>Ciudad</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCodigoPostal">
           <property name="text">
            <string>Código Postal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPais">
           <property name="text">
            <string>Pais</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="txtProveedor"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtCIF"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtTelefono"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtEmail"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDireccion"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtCiudad"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtCodigoPostal"/>
         </item>
         <item>
          <widget class="QLineEdit" name="txtPais">
           <property name="text">
            <string>España</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="cmdYN">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
